tests/github/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2020-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Tests package."""
```

Comment:
I remember seeing a similar package structure somewhere under `openfl`, but I don't understand very well why we need a separate `github` package (supposedly for tests running in the CI?). I suggest to simplify, and move the the `test_task_runner.py` module right under `tests/openfl_contrib`.

Reply:
Similar to my other reply. A lot of the structural choices was made to mirror openfl base repo. For the sake of this first PR, we'll keep it as is, but let's discuss how to adjust the structure (if needed)